### PR TITLE
rename transformer options

### LINKED ISSUE #14: switch entryPoint to entry_point
to match the argument style we have in all the other transformers, we should probably switch to use lower_case_with_underscores for these args.

### PR DESCRIPTION
**Pull request by jakemac53**
fixes https://github.com/dart-lang/initialize/issues/14

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
-## 0.1.1-dev
+## 0.2.0-dev
 
-Added `htmlEntryPoint` option to the transformer. This will search that file for
-any script tag whose src is `entryPoint` and rewrite it to point at the
-bootstrapped file `newEntryPoint`.
+* `entryPoint` and `newEntryPoint` transformer options were renamed
```
Comment (sigmundch) [inline CHANGELOG.md]: were renamed => were renamed to?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
-## 0.1.1-dev
+## 0.2.0-dev
 
-Added `htmlEntryPoint` option to the transformer. This will search that file for
-any script tag whose src is `entryPoint` and rewrite it to point at the
-bootstrapped file `newEntryPoint`.
+* `entryPoint` and `newEntryPoint` transformer options were renamed 
+`entry_point` and `new_entry_pont`.
+
+* Added `html_entry_point` option to the transformer. This will search that file
```
Comment (sigmundch) [inline CHANGELOG.md]: sad they can't detect these diffs correctly :-(  :-1

**Comment by sigmundch** [comment]
lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (jakemac53) [inline CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (jakemac53) [inline CHANGELOG.md]: lol ya, weird

**Pull request merged**